google/cloud/bigquery/client.py

```diff
@@ -891,6 +891,12 @@ def set_iam_policy(
 
         return Policy.from_api_repr(response)
 
+    def set_default_query_job_config(
```

Comment:
I recommend exposing this as an `@property` called `default_query_job_config`. Python style discourages Java-style get/set methods.

Reply:
Good to know. Fixed. Thanks